scripts/configure.py

```diff
@@ -715,10 +718,9 @@ def failed():
     if failed():
       return False
   if not web and restart_updater: # if web and restart_updater are True this restart happens in the _update_web function
```

Comment:
Since restart_updater is False by default, does this mean it won't restart during an update? Only from command-line usage of configure.py?

Reply:
Updater restart needs to be called after the update finishes. So it is called separately by the web updater

Reply:
The weird case here is that --restart-updater only reboots AmpliPi if it is called by itself if called with --web it just restarts both of the services.